scripts/artifacts/tag-rc2ga-on-repos.sh

```diff
@@ -44,6 +44,12 @@ function tag_oci_artifact_repos() {
     "tikv/pd/package"
     "tikv/tikv/package"
   )
+  # binlog is stoped releasing since v8.4.0.
```

Comment:
The comparison here may encounter an unexpected situation like v8.10.0 < v8.4.0.

Reply:
I fixed them.